lib/elixir/lib/macro.ex

```diff
@@ -172,6 +172,7 @@ defmodule Macro do
     * `:end_of_expression` - denotes when the end of expression effectively
       happens. Available for all expressions except the last one inside a
       `__block__` (when `:token_metadata` is true)
+    * `:indent` - The indentation of a non-interpolated sigil heredoc
```

Comment:
Would the correct name here be "indentation"? Since we are not asking it to indent but rather, returning the indentation? Maybe we should call it indentation in the EEx option as well for simplicity? Like "column".